### PR TITLE
fix(unlock-app): adding support for Timezones when setting up event metadata

### PR DESCRIPTION
**Pull request by julien51**
# Description

We are now supporting timezone to show accurate times on events.

# Checklist:

- [X] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: unlock-app/src/components/content/event/utils.tsx [hunk 1]
```diff
@@ -1,8 +1,16 @@
...
+import timezone from 'dayjs/plugin/timezone'
 
-export const getEventDate = (metadata: Partial<Metadata>): Date | null => {
-  if (metadata.ticket.event_start_date) {
-    return new Date(Date.parse(metadata.ticket.event_start_date.toString()))
+dayjs.extend(utc)
+dayjs.extend(timezone)
+
+export const getEventDate = (ticket: any): Date | null => {
+  if (ticket.event_start_date) {
```
Comment (kalidiagne) [inline unlock-app/src/components/content/event/utils.tsx:9]:
```suggestion
  if (ticket?.event_start_date && ticket?.event_start_time) {
```

File: unlock-app/src/components/interface/locks/metadata/index.tsx [hunk 2]
```diff
@@ -229,6 +229,7 @@ export function UpdateMetadataForm({ lockAddress, network, keyId }: Props) {
   const isLockSelected = selected.lockAddress && selected.network
 
   const isTokenURIEditable = useMemo(() => {
+    return true
```
Comment (kalidiagne) [inline unlock-app/src/components/interface/locks/metadata/index.tsx:232]: is this necessary?

File: unlock-app/src/components/content/event/utils.tsx (hunk 1 above)
Comment (julien51) [reply unlock-app/src/components/content/event/utils.tsx:9]: Thanks!

File: unlock-app/src/components/interface/locks/metadata/index.tsx (hunk 2 above)
Comment (julien51) [reply unlock-app/src/components/interface/locks/metadata/index.tsx:232]: oups! No it is just to make debugging easier!

Comment (julien51) [inline unlock-app/src/components/interface/locks/metadata/index.tsx:232]:
```suggestion
```

**Review by searchableguy — APPROVED** [review]

**Pull request merged**